### DISCUSSION TITLE
Generate synthetic data from an existing dataset

### DISCUSSION THREAD

**Discussion by AshlinHarris** [synthetichealth/synthea#1355] [General]
Can Synthea and Module Builder take an existing dataset as input and produces a statistically similar synthetic dataset?

The only forms of customization I could find involve adjusting [the properties](https://github.com/synthetichealth/synthea/blob/master/src/main/resources/synthea.properties) or providing [command-line flags](https://github.com/synthetichealth/synthea/discussions/1300).

**Comment by jawalonoski** [comment]
In short, "No, that is not how Synthea works."

The software assumes who aren't sitting on a mountain of real data, and have no access to real data.

We are working on strategies to take advantage of existing data in a safe and secure way, but there is no timeline and none of that capability is available.
